docs/markdown/Using Pants/validating-dependencies.md

```diff
@@ -156,6 +156,30 @@ __dependencies_rules__(
 # Due to the `extend=True` flag, which inherits the parent rules after those just declared.
 ```
 
+> 📘 Any rule globs using the declaration path anchoring mode that is inherited using `extend=True` will be anchored to the path of the current BUILD file, not the original one where the rule was extended from.
+
+Example show casing the caveat for `extend=True` using paths anchored to the declaration path (see [glob syntax](doc:targets#glob-syntax) for details on anchoring modes):
```

Comment:
"showcasing" is a single word. But also, do we need this paragraph at all, or would just "For example:" do, since the paragraph above puts us in the right context already?